pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -30,6 +30,59 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+// Returns the Console environment variables set in configuration.
+func consoleSecretEnvVars(t *miniov2.Tenant) []corev1.EnvFromSource {
+	envVars := []corev1.EnvFromSource{
+		{
+			SecretRef: &corev1.SecretEnvSource{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: t.Spec.Console.ConsoleSecret.Name,
+				},
+			},
+		},
+	}
+	return envVars
+}
+
+// Adds required Console environment variables
+func consoleEnvVars(t *miniov2.Tenant) []corev1.EnvVar {
+	envVars := []corev1.EnvVar{
+		{
+			Name:  "MINIO_SERVER_URL",
+			Value: t.MinIOServerEndpoint(),
+		},
+	}
+	if t.HasLogEnabled() {
+		envVars = append(envVars, corev1.EnvVar{
+			Name: miniov2.LogQueryTokenKey,
```

Comment:
This needs to be `MINIO_LOG_QUERY_AUTH_TOKEN` - we need to test full deployment once @nitisht

Reply:
yes, will test it once with everything enabled.